pkg/apis/core/validation/seed.go

```diff
@@ -248,15 +248,22 @@ func validateSeedNetworks(seedNetworks core.SeedNetworks, fldPath *field.Path, i
 	}
 
 	var (
-		primaryIPFamily = helper.DeterminePrimaryIPFamily(seedNetworks.IPFamilies)
-		networks        []cidrvalidation.CIDR
+		primaryIPFamily          = helper.DeterminePrimaryIPFamily(seedNetworks.IPFamilies)
+		networks                 []cidrvalidation.CIDR
+		reservedSeedServiceRange = cidrvalidation.NewCIDR(v1beta1constants.ReservedSeedServiceRange, field.NewPath(""))
 	)
 
 	if !inTemplate || len(seedNetworks.Pods) > 0 {
 		networks = append(networks, cidrvalidation.NewCIDR(seedNetworks.Pods, fldPath.Child("pods")))
 	}
 	if !inTemplate || len(seedNetworks.Services) > 0 {
-		networks = append(networks, cidrvalidation.NewCIDR(seedNetworks.Services, fldPath.Child("services")))
+		services := cidrvalidation.NewCIDR(seedNetworks.Services, fldPath.Child("services"))
+		networks = append(networks, services)
+		// Service range must not be larger than /8 for ipv4
+		if len(services.ValidateIPFamily(cidrvalidation.IPFamilyIPv4)) == 0 {
```

Comment:
Nit: Personally, I find the `if` condition hard to grasp as it implies the reader knows that the validate function returns a slice of errors and an empty list implies that the service range uses IPv4. Maybe a simple helper function could be less confusing.